third_party/ruby/build-ruby.bzl

```diff
@@ -224,10 +228,13 @@ def _build_ruby_impl(ctx):
             lib = libdir,
             share = sharedir,
             static_libs = static_libs,
+            static_lib_ruby = static_lib_ruby,
+            static_lib_ripper = static_lib_ripper,
         ),
         DefaultInfo(
             files = depset(outputs),
         ),
+        CcInfo(),
```

Comment:
What is the `CcInfo` provider used for?

Reply:
It's because `rubyfmt_static_deps_impl` depends on this. When a `cc_binary` targets `rubyfmt_static_deps`, it expects a `ccinfo` from that, but also from everything `rubyfmt_static_deps` depends on transitively, however, because we don't need anything specifically from this step, it's just a noop